### ISSUE TITLE
Script Trying to Parse Invalid URL

### ISSUE THREAD

**Issue by Prashanthvsdvn** [sdaqo/anipy-cli#127]
**Describe the bug**
Trying to open any episode, results in Invalid URL. I am not sure if its caused by some change in Gogo.

**To Reproduce**
1. Open script
2. Search for an series and Episode
4. Error

**Expected behavior**
Correct URL should be parsed and passed on to MPV

**Info**
```
Name: anipy-cli
Version: 2.7.19
Summary: Little tool in python to watch anime from the terminal (the better way to watch anime)
Home-page: https://github.com/sdaqo/anipy-cli
Author: sdaqo
Author-email: sdaqo.dev@protonmail.com
License: GPL-3.0
Location: C:\Python311\Lib\site-packages
Requires: beautifulsoup4, better-ffmpeg-progress, m3u8, moviepy, pycryptodomex, pypresence, python-dateutil, python-mpv, pyyaml, requests, setuptools, tqdm
Required-by:
```

**Platform**
 - OS: Windows 11

**Additional context**
Error log
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Python311\Scripts\anipy-cli.exe\__main__.py", line 7, in <module>
  File "C:\Python311\Lib\site-packages\anipy_cli\run_anipy_cli.py", line 8, in main
    cli.run_cli()
  File "C:\Python311\Lib\site-packages\anipy_cli\cli\cli.py", line 37, in run_cli
    cli_class(options=args, rpc_client=rpc_client).run()
  File "C:\Python311\Lib\site-packages\anipy_cli\cli\clis\base_cli.py", line 36, in run
    self.process()
  File "C:\Python311\Lib\site-packages\anipy_cli\cli\clis\default_cli.py", line 34, in process
    url_parser.stream_url()
  File "C:\Python311\Lib\site-packages\anipy_cli\url_handler.py", line 347, in stream_url
    enc_keys = self.get_enc_keys()
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\anipy_cli\url_handler.py", line 305, in get_enc_keys
    page = self.session.get(self.entry.embed_url).text
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\requests\sessions.py", line 600, in get
    return self.request("GET", url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\requests\sessions.py", line 573, in request
    prep = self.prepare_request(req)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\requests\sessions.py", line 484, in prepare_request
    p.prepare(
  File "C:\Python311\Lib\site-packages\requests\models.py", line 368, in prepare
    self.prepare_url(url, params)
  File "C:\Python311\Lib\site-packages\requests\models.py", line 445, in prepare_url
    raise InvalidURL(f"Invalid URL {url!r}: No host supplied")
requests.exceptions.InvalidURL: Invalid URL 'https:https://playtaku.online/streaming.php?id=MTk4NjI=&title=Initial+D%3A+First+Stage+Episode+1&typesub=SUB': No host supplied
```

**Comment by Prashanthvsdvn** [comment]
Duplicate of #125 Closing
